ephios/core/signup/methods.py

```diff
@@ -126,6 +127,37 @@ def reverse_event_detail(self, event):
         return event.get_absolute_url()
 
 
+@dataclasses.dataclass(frozen=True)
+class ForeignParticipant(AbstractParticipant):
```

Comment:
Name could be:

* Placeholder
* Mock
* Strawman
* Oneoff
* Unidentified
* Anonoumous

I think Placeholder and unidentified resemble it best. 
The participation is there to stand in for a person without an actual account/method of identification or correlation.